iina/InfoDictionary.swift

```diff
@@ -44,14 +50,57 @@ struct InfoDictionary {
     bundleIdentifier + ".build"
   }
 
+  /// The type of build used to generate this IINA executable.
+  ///
+  /// This corresponds to the Xcode build configuration.
+  var buildType: BuildType {
+    guard let buildConfiguration = buildConfiguration else { return .nightly }
+    return BuildType(rawValue: buildConfiguration) ?? .nightly
+  }
+
+  /// A string identifying the Xcode build configuration that was used to generate this executable.
+  ///
+  /// IINA's convention is that if there is no indication of the type of build then it is a release build. Therefore this property is `nil` if
+  /// this executable was built using the release configuration. Otherwise this property contains a string suitable for display to the user.
+  var buildTypeIdentifier: String? { buildType == .release ? nil : buildType.description }
+
   var bundleIdentifier: String { dictionary["CFBundleIdentifier"] as! String }
 
   var copyright: String { dictionary["NSHumanReadableCopyright"] as! String }
 
   let dictionary = Bundle.main.infoDictionary!
 
+  /// A Boolean value that indicates whether this executable was an optimized (not debug) build.
+  #if DEBUG
+  let isRelease = false
```

Comment:
I'm thinking this should be renamed `isDebug`. This is tied to the `Preprocessor Macros` Xcode setting. It is indicating the Debug Xcode build configuration was used.